static/src/javascripts/projects/common/modules/commercial/creatives/fabric-video.js

```diff
@@ -0,0 +1,104 @@
+define([
+    'qwery',
+    'common/utils/fastdom-promise',
+    'common/utils/detect',
+    'common/utils/template',
+    'text!common/views/commercial/creatives/fabric-video.html'
+], function (qwery, fastdom, detect, template, fluid250VideoBgStr) {
+    var fluid250VideoBgTpl;
+
+    return Fluid250VideoBg;
+
+    function Fluid250VideoBg(adSlot, params) {
+        var isUpdating = false;
+        var hasVideo, video, layer2, inView;
+
+        adSlot = adSlot instanceof HTMLElement ? adSlot : adSlot[0];
+        fluid250VideoBgTpl || (fluid250VideoBgTpl = template(fluid250VideoBgStr));
+
+        hasVideo = !detect.isIOS();
+        if (hasVideo) {
+            params.videoOrPoster = '<video webkit-playsinline muted class="creative__video creative__video--' + params.Videoalignment + '"><source src="' + params.VideoURL + '" type="video/mp4"></video>';
+        } else {
+            params.videoOrPoster = '<div class="poster" style="background-image:url(' + params.VideoBackupImage + ')"></div>';
+        }
+
+        params.Layer2clickbuttonMarkup = params.Layer2clickbutton ?
```

Comment:
In that case, yes. Normally you can include logic in templates, but I have experienced problems with the `text!` plugin with expressions like `<%if(xxx === 'test'){%>`. Tests for the truthy value of an object value works though. Thanks!